### ISSUE TITLE
Trade more than just equities

### ISSUE THREAD

**Issue by maxbbraun** [maxbbraun/trump2cash#5]
E.g. currency based on sentiment toward countries or governments? Futures while markets are closed?

**Comment by doodzik** [comment]
Trading [knockouts](http://www.investopedia.com/terms/k/knock-outoption.asp) instead of stocks could be an interesting way to leverage the movement in price.

What do you think?

**Comment by domingohui** [comment]
Total beginner when it comes to trading... Wondering if you want to liquidate your position at the end of day, would knockouts be a bit illiquid compared to the underlying stock?

**Comment by doodzik** [comment]
Yes, but as long as you have a small position and the bid-ask spread doesn't become too wide it shouldn't be a problem.

It would be interesting to see how trump affects the spread over time.

**Comment by matthazinski** [comment]
I'd also like the ability to trade in short-term volatility and index funds (e.g. when executive orders are issued/challenged/rescinded).

**Comment by AlainPilon** [comment]
@matthazinski talk about volatility highlight what I was about to say: instead of buy/short, the bot should use short lived options(calls/put). Buying or selling would be based on the expected volatility effect of the tweet.

Options offer better leverage and since most of the changes happen in a few days, we could use options with close expiration dates.
